profilepage/index.php

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>VITA Lincoln</title>
+	<?php
+		$root = realpath($_SERVER["DOCUMENT_ROOT"]);
+		require_once "$root/server/header.php";
+	?>
+	<!-- <link href="/assets/css/main.css" rel="stylesheet"> -->
+	<link href="/assets/css/form.css" rel="stylesheet">
+</head>
+<body>
+	<?php
+		//$page_subtitle = 'Create Volunteer Profile';
+		//require_once "$root/components/nav.php";
+	?>
+	<div>
+		<div class="container">
+			<div class="row justify-content-center">
+				<div class="col col-12 col-sm-8">
+					<div id="responsePlaceholder" style="display: none;"></div>
+					<form action="" autocomplete="off" class="cmxform" id="vitaProfileEdit" method="post" name="vitaProfileEdit">
+						<h2 class="form-title">Create Volunteer Profile</h2>
+
+						<div class="form-textfield">
+							<input class='' id="firstNameProfile" name="firstNameProfile" required="" type="text">
+							<span class="form-bar"></span>
+							<label class="form-label form-required" for="firstNameProfile">First Name</label>
+						</div>
+
+						<div class="form-textfield">
+							<input class='' id="lastNameProfile" name="lastNameProfile" required="" type="text">
+							<span class="form-bar"></span>
+							<label class="form-label vita-form-required" for="lastNameProfile">Last Name</label>
+						</div>
+
+						<div class="form-textfield">
+							<input class="" id="phoneProfile" name="phoneProfile" required="" type="tel">
+							<span class="form-bar"></span>
+							<label class="form-label form-required" for="phoneProfile">Phone Number</label>
+						</div>
+
+						<div class="form-textfield">
+							<input class="" id="emailProfile" name="emailProfile" required="" type="text">
+							<span class="form-bar"></span>
+							<label class="form-label form-required" for="emailProfile">Email</label>
+						</div>
+						<div class="form-group" style="margin-bottom: 10px;" required="">
+							<select id="languageSkills" class="form-control">
+								<option value='' disabled selected>Foreign Language</option>
+								<option value="None">None</option>
+								<option value="Spanish">Spanish</option>
+								<option value="Arabic">Arabic</option>
+								<option value="Vietnamese">Vietnamese</option>
+								<option value="Other">Other</option>
+							</select>
+						</div>
+						<div class="form-group" style="margin-bottom: 10px;" required="">
+					<label class="form-label form-required" for="taxSkills">Can you file taxes?</label>
+						<select id="taxSkills" class="form-control" onchange="taxFunction();" name="taxSkills">
+							<option value="No">No</option>
+							<option value="Yes">Yes</option>
+						</select>
+					</div>
+					<div class="form-group" style="margin-bottom: 10px;" id="skillType">
+					</div>
+						<div class="apptSelect form-group">
+						<h5>Choose Your Shift Times</h2>
+				     <select id="sbOne" multiple="multiple" class="form-control">
```

Comment:
This part here is really the most important part of this page. This must get done, and this must be done well. This is literally the only way that volunteers can sign up for times to work.